assets/src/blocks/Submenu/BackTop.js

```diff
@@ -0,0 +1,18 @@
+import useScrollPosition from '@react-hook/window-scroll';
+
+export const BackTop = ({}) => {
+  const cookies = document.getElementById('set-cookie');
+
+  const cookiesVisible = cookies && cookies.style.display !== 'none';
```

Comment:
This will not work... I was initially using `style.display` but found out that I had to use `getComputedStyle` instead... I think it's because the display property isn't set directly in the style object, but via css!